projects/kit/tokens/countries-masks.ts

```diff
@@ -217,4 +217,8 @@ export const TUI_COUNTRIES_MASKS = tuiCreateToken<Record<TuiCountryIsoCode, stri
     [TuiCountryIsoCode.ZA]: '+27##-###-####',
     [TuiCountryIsoCode.ZM]: '+260##-###-####',
     [TuiCountryIsoCode.ZW]: '+263#-######',
-});
+};
+
+export const TUI_COUNTRIES_MASKS = tuiCreateToken<Record<TuiCountryIsoCode, string>>(
```

Comment:
Let's add a helper like we do with other options (for example [here](https://github.com/taiga-family/taiga-ui/blob/main/projects/kit/components/checkbox/checkbox.options.ts#L26))